config/routes.rb

```diff
@@ -7,6 +7,15 @@
 
   # Defines the root path route ("/")
    root "users#index"
+   get 'posts/new', to: 'posts#new', as: 'new_posts'
+   post 'posts', to: 'posts#create'
+
+   get 'comments/new', to: 'comments#new', as: 'new_comments'
+   post 'comments', to: 'comments#create'
+
+  
+   post 'likes', to: 'likes#create'
```

Comment:
- [ ] Please kindly nest this routes as you rightly did for `user&post` inline 19. 👍  This will help to organize and express the relationship between users and their `posts/comments/likes` in a structured and meaningful way. ℹ️